README.md

```diff
@@ -51,6 +51,42 @@ This document provides the information needed to create a DASH packager that is
 
   See https://github.com/google/edash-packager/blob/master/CONTRIBUTING.md for details.
 
+# Using docker for testing/development #
+
+1. Requirements:
+
+   1. Clone this repo.
```

Comment:
Please include Dockerfile in the official repo and update build instruction accordingly. Thanks!